src/swarm/neo/client/ConnectionSet.d

```diff
@@ -356,6 +356,30 @@ public final class ConnectionSet : RequestOnConn.IConnectionGetter
 
     /***************************************************************************
 
+        Causes all connections to be dropped and re-established. This method is
+        only intended for use in tests.
+
+    ***************************************************************************/
+
+    public void reconnectAll ( )
+    {
+        for (auto conn = this.connections.getBoundary!(true)(); conn !is null;)
+        {
+            debug (SwarmConn)
+            {
+                auto address = conn.remote_address;
+                Stdout.formatln("ConnectionSet: reconnect {}:{}",
+                    address.address_bytes, address.port);
+            }
+
+            auto next = this.connections.getNext!(true)(conn);
+            conn.reconnect();
+            conn = next;
```

Comment:
You can't do `conn = this.connections.getNext!(true)(conn)` in the third param for `for` loop?

Reply:
Indeed, that's simpler. (I just cut & pasted from another method, but that looks like it has different concerns.) Updated.